examples/lock-app/nrf5/make_flasher.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import argparse
+import datetime
+import os
+import stat
+import sys
+import textwrap
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Flashing script generator.')
+
+    parser.add_argument("--output", metavar='FILENAME', required=True,
+                        help='flashing script name')
+    parser.add_argument("--image", metavar='FILENAME', required=True,
+                        help='image to be flashed')
```

Comment:
Should add a --flash-softdevice option probably!

And maybe one that does a full erase?

Reply:
Done